composer.json

```diff
@@ -11,10 +11,12 @@
             "email": "dcramer@gmail.com"
         }
     ],
+    "minimum-stability": "dev",
     "require-dev": {
         "friendsofphp/php-cs-fixer": "~2.1",
         "phpunit/phpunit": "^4.8 || ^5.0",
-        "monolog/monolog": "*"
+        "nokitakaze/testhttpserver": "^0.1.0",
```

Comment:
While I appreciate the work you've done to create a HTTP server, I can't really see why you've done this. Unit tests do not test that a server correctly processes something, they simulate the behaviour by mocking the response data...